src/gui/Src/Gui/CPURegistersView.cpp

```diff
@@ -542,6 +581,13 @@ void CPURegistersView::displayCustomContextMenuSlot(QPoint pos)
             wMenu.addAction(wCM_Modify);
         }
 
+        if(mINCREMENTDECREMET.contains(mSelected))

```

Comment:
Also I think these options should be all the way at the bottom, since they are used less frequently.

Reply:
IMHO, I think these options should be under "Modify value" because they do similar job which is changing registers value.

Reply:
You'd be surprised how often these options are used.